File: OddyseyServer/src/com/src/server/OperationData.java
package com.src.server;

import java.util.ArrayList;
import java.util.List;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlElementWrapper;
import javax.xml.bind.annotation.XmlRootElement;

import com.src.audio.AudioFile;
import com.src.login.User;

@XmlRootElement(name = "OperationData")
public class OperationData {
	@XmlElementWrapper(name = "AudioFiles")
	@XmlElement(name = "AudioFile")
	public List<AudioFile> audioFiles;
	@XmlElement(name = "SongToAdd")
	public AudioFile songToAdd;
	@XmlElement(name = "NameToDel")
	public String nameToDel;
	@XmlElement(name = "AuthorToDel")
	public String authorToDel;
	@XmlElement(name = "SongData")
	public String songData;
	@XmlElement(name = "SongToUpdate")
	public AudioFile songToUpdate;
	@XmlElement(name = "UserToAdd")
	public User userToAdd;
	@XmlElement(name = "UserToLog")
	public User userToLog;
	
	public OperationData() {
		audioFiles = new ArrayList<AudioFile>();
	}
	
}

File: Oddysey/UI/OddyseyUI/OddyseyUI/OperationData.cs
using System;
using System.Collections.Generic;
using System.Xml.Serialization;

namespace OddyseyUI
{
    [XmlRoot("OperationData")]
    public class OperationData
    {
        [XmlElement("AudioFiles")]
        public List<AudioFile> AudioFiles { get; set; }
        [XmlElement("SongToAdd")]
        public AudioFile SongToAdd { get; set; }
        [XmlElement("NameToDel")]
        public string NameToDel { get; set; }
        [XmlElement("AuthorToDel")]
        public string AuthorToDel { get; set; }
        [XmlElement("SongData")]
        public string SongData { get; set; }
        [XmlElement("SongToUpdate")]
        public AudioFile SongToUpdate { get; set; }
        [XmlElement("UserToAdd")]
        public User UserToAdd { get; set; }
        [XmlElement("UserToLog")]
        public User UserToLog { get; set; }
        [XmlElement("SongToFind")]  //Talvez lo simplifique
        public string SongToFind { get; set; }
        [XmlElement("ArtistToFind")]
        public string ArtisToFind { get; set; }
        [XmlElement("AlbumToFind")]
        public string AlbumToFind { get; set; }




        public OperationData()
        {
            AudioFiles = new List<AudioFile>();
        }
    }
}
